### ISSUE TITLE
[rawhide]: kdump fails to generate crash dump file after kernel crash

### ISSUE THREAD

**Issue by gursewak1997** [coreos/fedora-coreos-tracker#1320]
`kdump.crash` test fails to generate a vmcore file after the crash is triggered.
```
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 systemd[1]: Started kola-runext.service.
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 kola-runext-test.sh[1568]: + . /var/opt/kola/extdata/commonlib.sh
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 kola-runext-test.sh[1568]: ++ cmdline=($(</proc/cmdline))
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 kola-runext-test.sh[1568]: + case "${AUTOPKGTEST_REBOOT_MARK:-}" in
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 kola-runext-test.sh[1570]: ++ find /var/crash -type f -name vmcore
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 kola-runext-test.sh[1568]: + kcore=
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 kola-runext-test.sh[1568]: + test -z ''
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 kola-runext-test.sh[1568]: + fatal 'No kcore found in /var/crash'
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 kola-runext-test.sh[1568]: + echo 'No kcore found in /var/crash'
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 kola-runext-test.sh[1568]: No kcore found in /var/crash
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 kola-runext-test.sh[1568]: + exit 1
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 systemd[1]: kola-runext.service: Main process exited, code=exited, status=1/FAILURE
[2022-10-12T12:56:48.762Z] Oct 12 12:56:48 qemu0 systemd[1]: kola-runext.service: Failed with result 'exit-code'.
```
Recent rawhide build: [#376](https://jenkins-fedora-coreos-pipeline.apps.ocp.fedoraproject.org/blue/organizations/jenkins/build/detail/build/376/pipeline)
[console.txt](https://github.com/coreos/fedora-coreos-tracker/files/9775055/console.txt)
[journal.txt](https://github.com/coreos/fedora-coreos-tracker/files/9775058/journal.txt)

**Comment by gursewak1997** [comment]
Systemd package pin fixes the problem for now.
https://github.com/coreos/fedora-coreos-config/pull/2011

**Comment by dustymabe** [comment]
Were we able to file a bug on this?

**Comment by dustymabe** [comment]
There is a new systemd rc in rawhide. Can you test with that to see if it fixes the issue? 

https://bodhi.fedoraproject.org/updates/FEDORA-2022-a5f8e10002

**Comment by gursewak1997** [comment]
> There is a new systemd RC in `rawhide`. Can you test with that to see if it fixes the issue?
> 
> https://bodhi.fedoraproject.org/updates/FEDORA-2022-a5f8e10002

This did not fix the issue

**Comment by dustymabe** [comment]
> Also, I didn't file the issue yet as I thought we were missing information to add to the BZ but I will file the bug anyways.

If there's missing info then let's track it down (come to me if you have questions). If you want to share a draft of the BZ before you open it I'll be glad to review.

**Comment by gursewak1997** [comment]
Waiting on https://bugzilla.redhat.com/show_bug.cgi?id=2137631

**Comment by dustymabe** [comment]
This is resolved upstream by https://github.com/systemd/systemd/commit/a940f50

**Comment by dustymabe** [comment]
The BZ was closed and says `systemd-252.2-591.fc38` fixes the issue.

**Comment by gursewak1997** [comment]
The test passes in the latest rawhide [build](https://jenkins-fedora-coreos-pipeline.apps.ocp.fedoraproject.org/blue/organizations/jenkins/build/detail/build/623/pipeline) using `systemd-252.2-591.fc38`
